client/asset/btc/btc.go

```diff
@@ -238,6 +238,18 @@ type BTCCloneCFG struct {
 	// FeeEstimator provides a way to get fees given an RawRequest-enabled
 	// client and a confirmation target.
 	FeeEstimator func(RawRequester, uint64) (uint64, error)
+	// OmitAddressType causes the address type (bech32, legacy) to be omitted
+	// from calls to getnewaddress.
+	OmitAddressType bool
+	// LegacySignTxRPC causes the RPC client to use the signrawtransaction
+	// endpoint instead of the signrawtransactionwithwallet endpoint.
+	LegacySignTxRPC bool
+	// BooleanGetBlockRPC causes the RPC client to use a boolean second argument
+	// for the getblock endpoint, instead of Bitcoin's numeric.
+	BooleanGetBlockRPC bool
+	// SingularWallet signals that the node software supports only one wallet,
+	// so the RPC endpoint does not have a /wallet/{walletname} path.
+	SingularWallet bool
```

Comment:
`estimatesmartfee` also seems to have a different syntax, namely, no "mode":

```
2022-04-12 02:17:14.867 [ERR] CORE[doge]: Failed to get fee rate: rawrequest (estimatesmartfee) error: -1: estimatesmartfee nblocks

WARNING: This interface is unstable and may disappear or change!

Estimates the approximate fee per kilobyte needed for a transaction to begin
confirmation within nblocks blocks if possible and return the number of blocks
for which the estimate is valid. Uses virtual transaction size as defined
in BIP 141 (witness data is discounted).

Arguments:
1. nblocks     (numeric)

Result:
{
  "feerate" : x.x,     (numeric) estimate fee-per-kilobyte (in BTC)
  "blocks" : n         (numeric) block number where estimate was found
}

A negative value is returned if not enough transactions and blocks
have been observed to make an estimate for any number of blocks.
However it will not return a value below the mempool reject fee.

Example:
> dogecoin-cli estimatesmartfee 6
```

Reply:
Ah server has this set (`btc.BackendCloneConfig.FeeEstimator`), but not client.

Also, on server, the fee estimator never warmed up.  The following have been spamming logs for over 12 hrs:

```
2022-04-12 14:19:36.002 [ERR] DEX: Error retrieving fee rate for doge: fee could not be estimated
```

That is, no `err` back from `estimatefee`, just a rate of `<=0`.  On my machine is returns -1 (!?)

Actually, looks like client needs to use `estimatefee` instead of `estimatesmartfee`, which has that "unstable or may disappear or change" warning.

Maybe it always returns -1 or 0 because testnet does not have enough transactions.  In any case, I think we need to figure out a safe way to silence it.